### PR TITLE
Added tracking for clicked nav items.

### PR DESCRIPTION
**Pull request by fabianmarz**
- [Related Ticket](related-accessories)

### CODE REVIEW

File: assets/scripts/datalayer/common.js [hunk 1]
```diff
@@ -425,6 +427,32 @@ document.shopAnalytics = {
...
+    }
+    else {
+      context = 'unknown';
+    }
+
+    var event_data = {
+      'event': 'UniversalEvent',
+      'eventCategory': 'User Interaction | ' + context.toLowerCase() + ' menu',
+      'eventAction': 'Click',
+      'eventLabel': this.innerText.toLowerCase(),
```
Comment (fabianmarz) [inline assets/scripts/datalayer/common.js:451]: @daschmidt94, @LucaPipolo as menu items could be written inconsistently, e.g. `KITCHENS`, `my KITCHEN`, `My Kitchen` I decided to transform the text to all lowercase. The same applies for the aria label or ID attribute.

Also what value should be sent, if neither an aria-label or an ID attribute is set? Should we then prevent sending data at all?

File: assets/scripts/datalayer/common.js [hunk 2]
```diff
@@ -425,6 +427,32 @@ document.shopAnalytics = {
...
+    }
+    else {
+      context = 'unknown';
+    }
+
+    var event_data = {
+      'event': 'UniversalEvent',
+      'eventCategory': 'User Interaction | ' + context + ' menu',
+      'eventAction': 'Click',
+      'eventLabel': this.innerText,
```
Comment (fabianmarz) [inline assets/scripts/datalayer/common.js:451]: `innerText` returns the modified, and visible text. If the label transformed by CSS it's transformed as is. We could swap this for `textContent` which will return the (full) source code text.

As discussed with @daschmidt94, `innerText` and thus e.g. the all uppercase label is preferred as this makes it easier to identify which label was clicked.

A good summary can be read in [this SO answer](https://stackoverflow.com/a/35213639/2862127)

File: assets/scripts/datalayer/common.js [hunk 3]
```diff
@@ -192,7 +192,9 @@ document.shopAnalytics = {
     .on('click', '.products .product a', onProductClick)
     .on('click', '.single_add_to_cart_button', onProductAddToCart)
     .on('click', '.remove_from_cart_button, .woocommerce-cart-form .product-remove > a, .cart_item td.product-remove .remove', onRemoveSingleProduct)
-    .on('click', 'th.product-remove .remove', onEmptyCart);
+    .on('click', 'th.product-remove .remove', onEmptyCart)
```
Comment (LucaPipolo) [inline assets/scripts/datalayer/common.js:195]: Hm… can we be less specific with the selector? Ideally we are trying to use only classes, not tags like `th` to ensure future flexibility with the markup.

File: assets/scripts/datalayer/common.js [hunk 4]
```diff
@@ -192,7 +192,9 @@ document.shopAnalytics = {
...
+    .on('click', 'nav a', onNavItemClick)
```
Comment (LucaPipolo) [inline assets/scripts/datalayer/common.js:196]: Same same here.

File: assets/scripts/datalayer/common.js (hunk 3 above)
Comment (fabianmarz) [reply assets/scripts/datalayer/common.js:195]: I did not add this 🙈 I move the `;` to the next line so we can add new lines without the need to change two lines again.

File: assets/scripts/datalayer/common.js (hunk 4 above)
Comment (fabianmarz) [reply assets/scripts/datalayer/common.js:196]: As you can read from the ticket, the goal is to track click on nav item links. A consideration was to use the landmark `nav[role=navigation]`. Whenever you using `nav` the `role="navigation"` is implied already and thus redundant. Therefore I decided to use `nav` instead.

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Navigation_Role#Description

> Using the <nav> element will automatically communicate a section has a role of navigation. Developers should always prefer using the correct semantic HTML element over using ARIA.

**Pull request merged**